Marlin/src/lcd/dogm/status_screen_DOGM.cpp

```diff
@@ -371,9 +380,22 @@ FORCE_INLINE void _draw_centered_temp(const int16_t temp, const uint8_t tx, cons
 //
 FORCE_INLINE void _draw_axis_value(const AxisEnum axis, const char *value, const bool blink) {
   const AxisEnum a = TERN(LCD_SHOW_E_TOTAL, axis == E_AXIS ? X_AXIS : axis, axis);
-  const uint8_t offs = (XYZ_SPACING) * a;
-  lcd_put_wchar(X_LABEL_POS + offs, XYZ_BASELINE, axis_codes[axis]);
-  lcd_moveto(X_VALUE_POS + offs, XYZ_BASELINE);
+  #if ENABLED(STATUS_DISPLAY_INCHES)
+    if (parser.linear_unit_factor == 25.4f) {
```

Comment:
You need a better way to check for inches than repeating this `25.4f` constant in multiple places.
The parser already has `set_input_linear_units`, you should probably create a `get_input_linear_units` that returns the `LinearUnit` enum.

Reply:
Thank you for your input. I'm a self taught programmer, if you can even call me that.

I think what your trying to say is I need to make some sort of function that checks and call that function instead

Basically it checks and sets a bool so as to not mess with the float but once a cycle?

Reply:
It is more that I don't want to see 25.4f defined in multiple places. If it is in only one place, it is impossible for them to disagree.

Since there it already a `set_input_linear_units` is seems proper to add a function next to it to get the current state.

Reply:
I tried adding a function 'set_input_linear_units' but had an issue with variable types, so I went with a different approach for the time being. It built and tested fine on my hardware though.

I'll keep working on a separate function for the boolean if need be.

Reply:
@sjasonsmith I was able to get a separate function going for, but to me 21edaeb has easier to follow code. I can push the function code if you would like to see it.